Doc/votingFilesFormatsDoc.adoc

```diff
@@ -0,0 +1,88 @@
+Differents formats for _voting data_
+-----------------------------------
+
+Rank format
+~~~~~~~~~~~
+This format allows non strict preference. The rank of each alternative in the preference is filled in for every voter.
+
+.Example of Rank format
+[options="header"]
+|==============
+||1|2|3|4|5|6
+|1|1|1|5|5|4|1
+|2|3|1|4|4|4|1
+|3|2|2|2|2|1|1
+|4|3|4|3|3|3|2
+|5|4|3|1|1|2|3
+|==============
+
+.Meaning of Rank format
+[options="header"]
+|============================
+||Voter 1|Voter 2|Voter 3|Voter 4|Voter 5|Voter 6
+|Alternative 1|Voter 1 chose alternative 1 in 1st rank|1|5|5|4|Voter 6 chose alternative 1 in 1st rank (ex-aequo with alternatives 2 & 3)
+|Alternative 2|3|1|Voter 3 chose alternative 2 in 4th rank|4|4|1
+|Alternative 3|2|2|2|2|1|1
+|Alternative 4|3|4|3|3|3|2
+|Alternative 5|4|3|1|1|2|3
+|============================
+
+Profile format
+~~~~~~~~~~~~~~
+This format allows only strict preference. The preference for each voter is filled in.
+
+.Example of Profile format
+[options="header"]
+|==============
+|1|2|3|4|5|6
+|1|2|4|5|2|1
+|3|5|2|1|3|4
+|2|1|3|2|4|5
+|4|2|5|3|5|2
+|5|3|1|4|1|3
+|==============
+
+.Meaning of Profile format
+[options="header"]
+|==============
+|Voter 1|Voter 2|Voter 3|Voter 4|Voter 5|Voter 6
+|Alternative 1|2|4|Alternative 5|2|1
```

Comment:
Préciser : Alternative 1 is the preferred alternative of Voter 1 (ou qqch. d’équivalent) pour qu’on comprenne que les meilleures sont au-dessus.